src/nodes/chunk_dispatch/chunk_dispatch.h

```diff
@@ -76,6 +76,9 @@ typedef struct ChunkDispatchState
 	bool is_dropped_attr_exists;
 	int64 batches_decompressed;
 	int64 tuples_decompressed;
+
+	/* Should this INSERT be skipped due to ON CONFLICT DO NOTHING */
+	bool skip_insert;
```

Comment:
This is per-row, right? Might be more clear to mention it in the comment and maybe rename it to something like `skip_current_row`.

Reply:
renaming to skip_current_tuple